src/java.base/windows/classes/sun/nio/fs/WindowsPathParser.java

```diff
@@ -32,6 +32,13 @@
  */
 
 class WindowsPathParser {
+    private static final String LONG_PATH_PREFIX     = "\\\\?\\";
+    private static final String LONG_PATH_UNC_PREFIX = LONG_PATH_PREFIX + "UNC";
```

Comment:
Shouldn't this be "\\?\UNC\" ?

Reply:
Yes. Fixed in 90ef2d30a9ed8f7ddb5891836663e5f8728a948d.